Packs/AzureSentinel/Integrations/AzureSentinel/AzureSentinel.py

```diff
@@ -457,23 +459,32 @@ def test_module(client):
     return 'ok'
 
 
-def list_incidents_command(client, args, is_fetch_incidents=False):
-    filter_expression = args.get('filter')
+def list_incidents_command(client: AzureSentinelClient, args, is_fetch_incidents=False):
+    """Fetching incidents.
+    Args:
+        client: An AzureSentinelClient client.
+        args: Demisto args.
+        is_fetch_incidents: Is it part of a fetch incidents command.
+        first_fetch: Is it a first fetch.
+    Returns:
+        An array of incidents, and a latest created time.
+    """
+
     limit = None if is_fetch_incidents else min(200, int(args.get('limit')))
     next_link = args.get('next_link', '')
 
     if next_link:
         next_link = next_link.replace('%20', ' ')  # OData syntax can't handle '%' character
         result = client.http_request('GET', full_url=next_link)
+
     else:
-        url_suffix = 'incidents'
-        params = {
-            '$top': limit,
-            '$filter': filter_expression,
-            '$orderby': 'properties/createdTimeUtc asc'
-        }
+        params = {'$filter': args.get('$filter', ''),
```

Comment:
```suggestion
        params = {
            '$filter': args.get('filter', ''),
```
please stick to [pep8 style conventions](https://www.python.org/dev/peps/pep-0008/#indentation) (also verify your other changes)